src/NuGet.Core/NuGet.Protocol/VisualStudio/ExtensibleSourceRepositoryProvider.cs

```diff
@@ -1,6 +1,8 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+#if IS_DESKTOP
```

Comment:
The entire `NuGet.Protocol.VisualStudio` was built on `net45` only. At the same time I couldn't find direct dependencies of it in any other projects (besides some test project). The only way it was consumed via MEF. So I'd suggest merging its content into any of `NuGet.Clients` projects.